src/main/java/com/solr/clientwrapper/rest/SolrDocParseResource.java

```diff
@@ -0,0 +1,156 @@
+package com.solr.clientwrapper.rest;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+import java.net.URL;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.mysql.cj.xdevapi.JsonArray;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+
+
+
+
+@RestController
+@RequestMapping("/file")
+public class SolrDocParseResource {
```

Comment:
This is Sprint 5's work right? Which is done by me in Sprint 6?

Reply:
so should i delete this?

Reply:
No Document Parse by multipart file is done By Omkar

Reply:
Most probably.
If this is not used anywhere else